src/popup/pages/SeedPhraseVerifySettings.vue

```diff
@@ -57,12 +57,17 @@
         </div>
         <FixedScreenFooter>
           <BtnMain
+            v-if="hasError || !showNotification"
             class="verify-button"
-            :disabled="!selectedWordIds || selectedWordIds.length !== mnemonicShuffled.length"
+            :disabled="isVerifyButtonDisabled"
+            :text="$t('pages.seedPhrase.verify')"
             @click="verifyLastStep"
-          >
-            {{ $t('pages.seedPhrase.verify') }}
-          </BtnMain>
+          />
+          <BtnMain
+            v-else
+            :text="$t('common.backToHome')"
+            :to="{name: ROUTE_ACCOUNT, replace: true}"
```

Comment:
Why `replace: true` is used here?

Reply:
The reason `replace: true` is used, is because otherwise Ionic will cache the page.
This poses a security risk since if the user verifies the seed-phrase and then goes to the dashboard, if he leaves his device unattended someone could just go back and see the seed-phrase.

But on second thought, someone could see the seed-phrase anyway from the settings. I will remove the `replace: true`